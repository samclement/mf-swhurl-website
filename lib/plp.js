module.exports.parseNavigation = navigation => {
  return navigation.map(item => {
    const { name, url } = item
    return { name, url }
  })
}

module.exports.parseSearchResults = searchResults => {
  return {
    categoryFacets: parseCategoryFacets(searchResults.categoryNavData),
    facets: parseFacets(searchResults.facets),
    results: parseResults(searchResults.results)
  }
}

function parseResults(results) {
  return results.map(parseProductResult)
}

function parseCategoryFacets(cFacets) {
  return cFacets.map(parseCategoryFacet)
}

function parseFacets(facets) {
  return facets
    .filter(facet => {
      return (
        facet.code === 'allDesigners' ||
        facet.code === 'refinementColour' ||
        facet.code === 'clothesSize'
      )
    })
    .map(parseFacet)
}

function parseCategoryFacet(cFacet) {
  const parsed = {
    name: cFacet.name,
    isSelected: cFacet.active,
    url: cFacet.url,
    categories: []
  }
  if (cFacet.active && cFacet.categories) {
    parsed.categories = cFacet.categories.map(parseCategoryFacet)
  }
  return parsed
}

function parseFacet(facet) {
  return {
    name: facet.name,
    values: facet.values.map(parseFacetValue)
  }
}

function parseFacetValue(facetValue) {
  return {
    name: facetValue.name,
    url: facetValue.query.url
  }
}

function parseProductResult(product) {
  return {
    name: product.name,
    url: product.url,
    code: product.code,
    price: product.price.value,
    imageCount: product.galleryImageMaps.length,
    sizes: product.sizeList.map(size => {
      return parseSize(size)
    })
  }
}

function parseSize(size) {
  return {
    code: size.code,
    baseCode: size.baseCode,
    description: size.value,
    inStock: size.available
  }
}
