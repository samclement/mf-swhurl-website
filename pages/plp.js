import React, { useState, useEffect } from 'react'
import ProductListItem from '../components/blocks/ProductListItem'
import Nav from '../components/blocks/Nav'
import GenderHeader from '../components/elements/GenderHeader.js'
import FilterBlock from '../components/blocks/FilterBlock'
import Head from 'next/head'
import { mens, womens } from '../navigation.js'
import { getSearchResults } from '../services'
import styled from 'styled-components'
import CheeseburgerMenu from 'cheeseburger-menu'

const ProductsList = styled.div`
  display: flex;
  flex-flow: row wrap;
  justify-content: center;
`

const SearchContainer = styled.div`
  display: flex;
  align-items: flex-start;
`

const FilterBlockContainer = styled.div`
  display: flex;
  flex-flow: column nowrap;
`

const DesktopFilterBlockContainer = styled(FilterBlockContainer)`
  @media only screen and (max-width: 640px) {
    display: none;
  }
`

const FilterButton = styled.button`
  @media only screen and (min-width: 640px) {
    display: none;
  }
`

function Plp(props) {
  const { gender, results, categoryFacets, facets } = props
  const [menuOpen, setMenuOpen] = useState(false)
  useEffect(() => {
    document.body.style.overflow = menuOpen ? 'hidden' : 'auto'
  })
  return (
    <div>
      <Head>
        <title>mf.swhurl.com | {gender}</title>
      </Head>
      <Nav items={mens} gender="mens" />
      <Nav items={womens} gender="womens" />
      <GenderHeader gender={gender} />
      <FilterButton onClick={() => setMenuOpen(!menuOpen)}>
        Show Filters
      </FilterButton>
      <SearchContainer>
        <CheeseburgerMenu
          isOpen={menuOpen}
          width={220}
          innerClassName="touch"
          closeCallback={() => setMenuOpen(false)}
        >
          <FilterBlockContainer>
            <FilterBlock title="Category" filters={categoryFacets} />
            {facets.map((f, i) => {
              return <FilterBlock key={i} title={f.name} filters={f.values} />
            })}
          </FilterBlockContainer>
        </CheeseburgerMenu>
        <DesktopFilterBlockContainer>
          <FilterBlock title="Category" filters={categoryFacets} />
          {facets.map((f, i) => {
            return <FilterBlock key={i} title={f.name} filters={f.values} />
          })}
        </DesktopFilterBlockContainer>
        <ProductsList>
          {results.map((item, i) => {
            return <ProductListItem key={i} item={item} />
          })}
        </ProductsList>
      </SearchContainer>
    </div>
  )
}

Plp.getInitialProps = async ({ req, query }) => {
  const q = req && req.query ? req.query : query
  const res = await getSearchResults(q, !!req)
  const { results, categoryFacets, facets } = res.data
  const gender = q.url.match(/\/mens/) ? 'mens' : 'womens'
  return { results, gender, categoryFacets, facets }
}

export default Plp
