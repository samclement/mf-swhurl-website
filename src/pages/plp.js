import Plp from '../components/containers/Plp'

export default Plp
