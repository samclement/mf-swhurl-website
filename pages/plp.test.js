/* eslint-env jest */

import React from 'react'
import { render } from 'enzyme'
import 'jest-styled-components'
import Plp from './plp.js'
import { parseSearchResults } from '../lib/plp.js'
const data = require('../test-data/activewear.json')

describe('With Snapshot Testing', () => {
  it('Plp page shows "mf.swhurl.com" H1', () => {
    const r = { gender: 'mens', results: parseSearchResults(data).results }
    const wrapper = render(<Plp results={r.results} gender={r.gender} />)
    expect(wrapper).toMatchSnapshot()
  })
  it('Plp getInitialProps', async () => {
    const res = await Plp.getInitialProps({ query: { url: '/mens/' } })
    expect(res).toMatchSnapshot()
  })
  it('Plp getInitialProps', async () => {
    const res = await Plp.getInitialProps({ req: { query: { url: '/womens/' } } })
    expect(res).toMatchSnapshot()
  })
  it('Plp getInitialProps', async () => {
    const res = await Plp.getInitialProps({ req: { query: { url: '/mens/' } } })
    expect(res).toMatchSnapshot()
  })
})
