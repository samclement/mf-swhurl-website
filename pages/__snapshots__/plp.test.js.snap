// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Plp getInitialProps 1`] = `
Object {
  "categoryFacets": undefined,
  "facets": undefined,
  "gender": "mens",
  "results": undefined,
}
`;

exports[`With Snapshot Testing Plp getInitialProps 2`] = `
Object {
  "categoryFacets": undefined,
  "facets": undefined,
  "gender": "womens",
  "results": undefined,
}
`;

exports[`With Snapshot Testing Plp getInitialProps 3`] = `
Object {
  "categoryFacets": undefined,
  "facets": undefined,
  "gender": "mens",
  "results": undefined,
}
`;

exports[`With Snapshot Testing Plp page shows "mf.swhurl.com" H1 1`] = `
.c11 {
  width: 150px;
  padding: 10px;
  text-align: center;
}

.c2 {
  color: black;
  padding-right: 5px;
}

.c2:hover {
  color: #F00;
}

.c2:visited {
  color: black;
}

.c1 {
  font-weight: bold;
  text-transform: capitalize;
  padding-right: 10px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c6 {
  width: 170px;
  padding: 10px;
}

.c7 {
  padding: 0;
  margin: 0;
}

.c8 {
  list-style: none;
  padding-left: 0;
  margin-left: 0;
}

.c9 {
  color: black;
  font-size: 11px;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c10 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c3 {
  text-transform: capitalize;
  margin-bottom: 0px;
}

.c4 {
  align: left;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column nowrap;
  -ms-flex-flow: column nowrap;
  flex-flow: column nowrap;
}

<div>
  <nav
    class="c0"
  >
    <div
      class="c1"
    >
      mens
    </div>
    <a
      class="c2"
      href="/mens/shop/accessories"
    >
      Accessories
    </a>
    <a
      class="c2"
      href="/mens/shop/shoes"
    >
      Shoes
    </a>
    <a
      class="c2"
      href="/mens/shop/clothing"
    >
      Clothing
    </a>
    <a
      class="c2"
      href="/mens/shop/bags"
    >
      Bags
    </a>
    <a
      class="c2"
      href="/mens/shop/homeware"
    >
      Homeware
    </a>
  </nav>
  <nav
    class="c0"
  >
    <div
      class="c1"
    >
      womens
    </div>
    <a
      class="c2"
      href="/womens/shop/accessories"
    >
      Accessories
    </a>
    <a
      class="c2"
      href="/womens/shop/bags"
    >
      Bags
    </a>
    <a
      class="c2"
      href="/womens/shop/clothing"
    >
      Clothing
    </a>
    <a
      class="c2"
      href="/womens/shop/shoes"
    >
      Shoes
    </a>
    <a
      class="c2"
      href="/womens/shop/homeware"
    >
      Homeware
    </a>
  </nav>
  <h1
    class="c3"
  >
    mens
  </h1>
  <div
    class="c4"
  >
    <div
      class="c5"
    >
      <div
        class="c6"
      >
        <h2>
          Category
        </h2>
        <ul
          class="c7"
        >
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                checked=""
                readonly=""
                type="checkbox"
              />
               Activewear
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Blazers
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Casual Shirts
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Coats and Jackets
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Formal Shirts
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Jeans
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Knitwear
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Polo Shirts
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Shorts
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Suits
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Sweats
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Swimwear
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               T-Shirts
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Trousers
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Underwear and Nightwear
            </a>
          </li>
        </ul>
      </div>
      <div
        class="c6"
      >
        <h2>
          Designer
        </h2>
        <ul
          class="c7"
        >
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               1017 ALYX 9SM
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               2XU
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               And Wander
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Aztech Mountain
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Blackbarrett by Neil Barrett
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Caf√© Du Cycliste
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Castore
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Falke
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Fendi
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Helly Hansen
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Iffley Road
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               LNDR
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Mammut Delta X
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Moncler
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Moncler Genius
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Moncler Grenoble
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Nemen
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Peak Performance
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Perfect Moment
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Salomon
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Satisfy
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               The Upside
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Tilak
            </a>
          </li>
        </ul>
      </div>
      <div
        class="c6"
      >
        <h2>
          Colour
        </h2>
        <ul
          class="c7"
        >
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Black
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Blue
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Brown
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Green
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Grey
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Metallic
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Navy
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Neutral
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Orange
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Print
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Purple
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Red
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Tan
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               White
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               Yellow
            </a>
          </li>
        </ul>
      </div>
      <div
        class="c6"
      >
        <h2>
          Clothes Size
        </h2>
        <ul
          class="c7"
        >
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               XXS
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               XS
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               S
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               M
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               L
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               XL
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               XXL
            </a>
          </li>
          <li
            class="c8"
          >
            <a
              class="c9"
              href="#"
            >
              <input
                readonly=""
                type="checkbox"
              />
               3XL
            </a>
          </li>
        </ul>
      </div>
    </div>
    <div
      class="c10"
    >
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245717_1_small.jpg"
        />
        Hooded zip-up track jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263536_1_small.jpg"
        />
        Agile technical ripstop jacket 
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245726_1_small.jpg"
        />
        Logo-print cotton-blend T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261981_1_small.jpg"
        />
        Accelerate compression stretch leggings
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256827_1_small.jpg"
        />
        Hooded technical jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256819_1_small.jpg"
        />
        Hybrid mesh T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245719_1_small.jpg"
        />
        Technical track pants
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1247264_1_small.jpg"
        />
        Tech-Preme logo-patch technical sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256167_1_small.jpg"
        />
        FF-logo zip-through cotton-blend hooded sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256820_1_small.jpg"
        />
        Logo-patch raglan jersey T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256815_1_small.jpg"
        />
        Belted stretch trousers
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256180_1_small.jpg"
        />
        FF cotton-blend hooded sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261985_1_small.jpg"
        />
        GHST two-in-one jacket 
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1221480_1_small.jpg"
        />
        Logo-patch crew-neck cotton T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261994_1_small.jpg"
        />
        Urban jersey track pants
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256197_1_small.jpg"
        />
        Run West moth-eaten cotton T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1194379_1_small.jpg"
        />
        Contrast-panel hooded bomber jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256190_1_small.jpg"
        />
        Cult moth-eaten cotton tank top
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1221482_1_small.jpg"
        />
        Striped cotton-blend track pants
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1262681_1_small.jpg"
        />
        Rix hooded jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256166_1_small.jpg"
        />
        Front-stripe logo-printed jersey sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245725_1_small.jpg"
        />
        Logo print cotton-blend T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245720_1_small.jpg"
        />
        Zip-pocket track shorts
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245727_1_small.jpg"
        />
        Striped cotton-blend jersey T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263588_1_small.jpg"
        />
        The Redford cotton sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256157_1_small.jpg"
        />
        Baldwin checked cotton shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1247647_1_small.jpg"
        />
        Seamless compression tights
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256192_1_small.jpg"
        />
        Reverse cotton T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261935_1_small.jpg"
        />
        Marlow technical-twill running jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263586_1_small.jpg"
        />
        Big Logo-print cotton tank top
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261936_1_small.jpg"
        />
        Windsor II stretch-jersey leggings
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263540_1_small.jpg"
        />
        Sense Pro crew-neck T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245722_1_small.jpg"
        />
        Mesh zip-up track top
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263591_1_small.jpg"
        />
        The Newman cotton T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1245728_1_small.jpg"
        />
        Court-stripe leggings
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263596_1_small.jpg"
        />
        Big Logo-print cotton tank top
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261987_1_small.jpg"
        />
        GHST contrast-panel performance top
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263246_1_small.jpg"
        />
        Marion stretch-jersey T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261948_1_small.jpg"
        />
        Royston track pants
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1262688_1_small.jpg"
        />
        Fellows long-sleeved stretch-mesh T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256196_1_small.jpg"
        />
        Quarter-zip technical hooded jacket 
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256816_1_small.jpg"
        />
        Logo-print longline performance T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261980_1_small.jpg"
        />
        Compression running leggings
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1288573_1_small.jpg"
        />
        FF-logo appliqu√© cotton T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256818_1_small.jpg"
        />
        Mesh-panel performance top
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1247641_1_small.jpg"
        />
        Performance T-shirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1256811_1_small.jpg"
        />
        Climbing belted trousers
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261151_1_small.jpg"
        />
        Guard technical jacket 
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261142_1_small.jpg"
        />
        Constrast-panel performance jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261139_1_small.jpg"
        />
        Guard reflective jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261140_1_small.jpg"
        />
        Revolt technical jacket 
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261149_1_small.jpg"
        />
        Dondi crew-neck technical sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261148_1_small.jpg"
        />
        Scramble nylon shorts 
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261147_1_small.jpg"
        />
        Crushed technical fabric trousers
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261141_1_small.jpg"
        />
        Utility shell sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261145_1_small.jpg"
        />
        Dondi technical-shell sweatshirt
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1261143_1_small.jpg"
        />
        Marking reflective jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263534_1_small.jpg"
        />
        Haloes hybrid hooded down-filled jacket
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263551_1_small.jpg"
        />
        Agile mesh-panel performance shorts
      </div>
      <div
        class="c11"
      >
        <img
          src="//assetsprx.matchesfashion.com/img/product/1263537_1_small.jpg"
        />
        Agile polka dot-print performance jacket
      </div>
    </div>
  </div>
</div>
`;
